modules/statics/src/account.ts

```diff
@@ -183,6 +187,28 @@ export class HederaCoin extends AccountCoinToken {
   }
 }
 
+/**
+ * The Algo network supports tokens (assets)
+ * Algo tokens work similar to native ALGO coin, but the token name is determined by the chain,
+ * and the unique asset id on the chain: (t)algo:<assetId>
```

Comment:
[np] This doesn't really describe what the class does and it worded confusingly.